webapp/tabula_web.rb

```diff
@@ -92,6 +92,155 @@ def upload(file)
   return [job_batch, file_id]
 end
 
+def list_templates
```

Comment:
Hey @jeremybmerrill 

Can we move this to the new `Tabula::Workspace` class (in my PR)? I'd love to include this feature in our next release.